tests/integration/features/bootstrap/CoreContext.php

```diff
@@ -0,0 +1,930 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use GuzzleHttp\Client;
+use GuzzleHttp\Message\ResponseInterface;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+/**
+ * Features context.
+ */
+class CoreContext implements Context, SnippetAcceptingContext {
```

Comment:
oh - by simply copy-it-over I did mean to copy those methods which are needed :see_no_evil: